mempool/src/lib.rs

```diff
@@ -36,6 +40,7 @@ struct MempoolState {
     transactions_by_sender: HashMap<Address, BTreeSet<Arc<Transaction>>>,
     transactions_by_recipient: HashMap<Address, BTreeSet<Arc<Transaction>>>,
     transactions_sorted_fee: BTreeSet<Arc<Transaction>>, // sorted by fee, ascending
+    filter: filter::MempoolFilter,
```

Comment:
Use `use crate::filter::MempoolFilter;` to improve readability.
That should let you remove all the `filter::` prefixes in this file.